frontend/src/features/feedback/StackedCandidatesPaper.tsx

```diff
@@ -34,15 +36,38 @@ const StackedCandidatesPaper = ({
   ratings,
   recommendations,
 }: Props) => {
+  const [sortingCriteria, setSortingCriteria] = useState('be_president');
   const { t } = useTranslation();
-  const { baseUrl } = useCurrentPoll();
+  const { baseUrl, getCriteriaLabel } = useCurrentPoll();
+
+  const allCriteria = new Set(
+    recommendations.flatMap((entityScore) =>
+      entityScore.criteria_scores.map((cs) => cs.criteria)
+    )
+  );
 
   const nComparisons = Object.fromEntries(
     ratings.map((rating) => {
       return [rating.entity.uid, rating.n_comparisons];
     })
   );
 
+  const sortedRecommendations = recommendations
+    .filter(
+      (entityWithScores) =>
+        entityWithScores.criteria_scores.find(
+          ({ criteria }) => criteria == sortingCriteria
+        )?.score
```

Comment:
entities with score 0 should not be excluded:

```suggestion
      (entityWithScores) =>
        entityWithScores.criteria_scores.find(
          ({ criteria }) => criteria == sortingCriteria
        )?.score != null
```

Reply:
Thanks!!